File: Api/Api/Models/Personnel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Api.Models
{
    public class Personnel
    {
        [Key]
        public int id { get; set; }
        public string name { get; set; }
        public string email { get; set; }
        public string phone { get; set; }
    }
}

File: My App/app/model/Personnel.js
Ext.define('MyApp.model.Personnel', {
    extend: 'Ext.data.Model',
    //alias: 'model.Personnel',

    fields: [
        'id','name', 'email', 'phone'
    ],

    proxy: {
        method: 'POST',
        type: 'rest',
        url : 'http://localhost:25231/api/Personnel/'
    }
});
